### ISSUE TITLE
python package problem

### ISSUE THREAD

**Issue by MeaninglessAI** [rikiblu/8-puzzle---State-Space-Searches#1]
Hi, thanks for you code, it's helpful. But when I run the code, I encounted the error "ModuleNotFoundError: No module named 'Queue'“ . but the "Queue"  seems  not a python package.
Looking forward to your reply!

**Comment by rmssoares** [comment]
Hey, Jinrui Yang! Are you using Python 3 or Python 2? This code, I believe, was written in Python 2, which might cause some issues. I might update the code later on.
https://docs.python.org/2/library/queue.html (the Queue in question)
If necessary, try updating the "Queue" to "queue" in the code to adhere with python 3!
________________________________

**Comment by MeaninglessAI** [comment]
Thank you for your prompt reply , I just used python3. when I  change my env to python 2, it works !
But I just tried to change the "Queue" to "queue" in python 3 , it still  seems  error. maybe it's grammar problem caused by  the different python version.  like this below:

![image](https://user-images.githubusercontent.com/23152242/67672631-590d0b00-f9b3-11e9-8dfb-1d155c3933f9.png)

Anyway, it works fine for python 2.7. thank you for your helpful reply. :)
